sonar-ws/src/main/gen-java/org/sonarqube/ws/Rules.java

```diff
@@ -1139,11 +1108,10 @@ public final boolean isInitialized() {
 
     public void writeTo(com.google.protobuf.CodedOutputStream output)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)